components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -71,7 +71,9 @@
     // OIDC related properties
     boolean isRequestObjectSignatureValidated = Boolean.parseBoolean(request.getParameter("validateRequestObjectSignature"));
     boolean isIdTokenEncrypted = Boolean.parseBoolean(request.getParameter("encryptIdToken"));
-    
+    String encryptionAlgorithm = request.getParameter("encryptionAlgorithm");
```

Comment:
Can't we use the constants defined in OAuthConstants.java for encryptionAlgorithm & encryptionMethod

Reply:
These are actually the name tags defined in JSP files. I suppose we could import the constant class into the JSPs and use the constants but I don't see any actual use in that. Common practice is to use these as it is. WDYT?